docs/installation.md

```diff
@@ -106,13 +106,13 @@ XO needs the following packages to be installed. Redis is used as a database by
 For example, on Debian/Ubuntu:
 
 ```sh
-apt-get install build-essential redis-server libpng-dev git python3-minimal libvhdi-utils lvm2 cifs-utils nfs-common
+apt-get install build-essential redis-server libpng-dev git python3-minimal libvhdi-utils lvm2 cifs-utils nfs-common ntfs-3g dmidecode
```

Comment:
I would only add them if they are necessary. 
If not I would only add a text on optional/useful packages